pkg/odo/cli/pipelines/ui/ui.go

```diff
@@ -0,0 +1,246 @@
+package ui
+
+import (
+	"gopkg.in/AlecAivazis/survey.v1"
+
+	"github.com/openshift/odo/pkg/odo/cli/ui"
+)
+
+// EnterGitRepo allows the user to specify the git repository in a prompt
+func EnterGitRepo() string {
+	var path string
+	var prompt *survey.Input
+	prompt = &survey.Input{
+		Message: "Provide the URL for your GitOps repository",
+		Help:    "The GitOps repository stores your GitOps configuration files, including your Openshift Pipelines resources for driving automated deployments and builds.  Please enter a valid git repository e.g. https://github.com/example/myorg.git",
+	}
+
+	err := survey.AskOne(prompt, &path, survey.Required)
+	ui.HandleError(err)
+
+	return path
+}
+
+// EnterInternalRegistry allows the user to specify the internal registry in a prompt
+func EnterInternalRegistry() string {
+	var path string
+	var prompt *survey.Input
+	prompt = &survey.Input{
+		Message: "Host-name for internal image registry e.g. docker-registry.default.svc.cluster.local:5000, used if you are pushing your images to the internal image registry",
+		Default: "image-registry.openshift-image-registry.svc:5000",
+	}
+
+	err := survey.AskOne(prompt, &path, nil)
+	ui.HandleError(err)
+
+	return path
+}
+
+func EnterImageRepoInternalRegistry() string {
+	var path string
+	var prompt *survey.Input
+	prompt = &survey.Input{
+		Message: "Image repository of the form <project>/<app> which is used to push newly built images.",
+		Help:    "By default images are built from source, whenever there is a push to the repository for your service source code and this image will be pushed to the image repository specified in this parameter, if the value is of the form <registry>/<username>/<repository>, then it assumed that it is an upstream image repository e.g. Quay, if its of the form <project>/<app> the internal registry present on the current cluster will be used as the image repository.",
+	}
+
+	err := survey.AskOne(prompt, &path, survey.Required)
+	ui.HandleError(err)
+
+	return path
+}
+
+func EnterDockercfg() string {
+	var path string
+	var prompt *survey.Input
+	prompt = &survey.Input{
+		Message: "Path to config.json which authenticates image pushes to the desired image registry, the default is <insert default>?",
```

Comment:
we can remove `<insert default>` here since the default is already added

Reply:
That was included in the documentation @chetan-rns , it was sugessted to be included there!

Reply:
Yeah. We need to show the default. But it's displayed twice
```
? Name of the Sealed Secrets Services that encrypts secrets <sealed-secrets-controller> [? for help] (sealed-secrets-controller) 
```

Reply:
got it.

Reply:
Solved, replaced with values.